common-lisp-parachute/README.md

```diff
@@ -0,0 +1,79 @@
+# gilded-rose
+
+## Setup
+
+### Install quicklisp
+
+To run this project install quicklisp (if not already done): 
+ - download https://beta.quicklisp.org/quicklisp.lisp
+ - load it with your common lisp implementation. The example with sbcl: 
+   > sbcl --load quicklisp.lisp
+ - run the command 
+   > (quicklisp-quickstart:install) 
+   in your common lisp implementation
+ - run the command 
+   > (ql:add-to-init-file) 
+   in your common lisp implementation
+
+### Install project
+Copy the project-folder containing this file into /quicklisp/local-projects/ that has been created when installing quicklisp.
+This is the root directory for quicklisp to search for the gilded-rose.asd file which defines the system (project) and its dependencies.
+The quicklisp-folder is usually created in your home-directory.
+
+### Work with the project
+
+Now you can load the project with 
+> (ql:quickload "gilded-rose") 
+in your common lisp-implementation and run the tests with 
+> (asdf:test-system "gilded-rose")
+.
+
+You can stub and mock functions and methods with the cl-mock-library which is already included in the system definition of the test-system:
+
+(with-mocks ()
+  (answer <your-method> (call-previous))
+  (<your-method> <argument-list>)
+  (is <your-testcase> (invocations '<your-method>)))
+
+## Documentation
```

Comment:
Instead of including this text here, I suggest instead linking to the requirements file in the folder above